### PR TITLE
Update README.md and Jazzy annotations

### PR DESCRIPTION
**Pull request by seabaylea**
Update the Jazzy annotations to include HTTPServer and remove the PoCSocket code, and add a proper README.md that provides some basic getting started information.

If/when this is merged I'll update the gh-pages branch with the generated docs.

### CODE REVIEW

File: Sources/HTTP/HTTPServer.swift [hunk 1]
```diff
@@ -28,21 +28,26 @@ public protocol HTTPServing : class {
 public class HTTPServer: HTTPServing {
     private let server = PoCSocketSimpleServer()
 
+    /// Create an instance of the server. This needs to be followed with a call to `start()`
     public init() {
     }
 
+    /// Start the HTTP server on the given `port`, using `handler` to process incoming requests
```
Comment (adellibovi) [inline Sources/HTTP/HTTPServer.swift:35]: I would add `- Parameter` too for port and handler, what do you think?

Comment (adellibovi) [reply Sources/HTTP/HTTPServer.swift:35]: I saw that in some other place of the code it's missing, if it's okay I would raise a PR fixing the missing docs for parameters.

**Review by adellibovi — CHANGES_REQUESTED** [review]

**Comment by seabaylea** [comment]
Sure - if the of the change in this PR looks ok then I'll merge and you can add the parameter descriptions (without risking merge conflicts).

**Review by adellibovi — APPROVED** [review]
Cool 👍  😄

**Comment by seabaylea** [comment]
Thanks @adellibovi. Merged.

**Pull request merged**